src/norm/pragmas.nim

```diff
@@ -16,3 +16,8 @@ template ro* {.pragma.}
 
   ``id`` field is ``ro`` by default.
   ]##
+
+template fk*(val: typed) {.pragma.}
+  ##[ Mark field as foreign key another type. ``val`` is either a type or a "type.field"
+  expression. If a type is provided, its ``id`` field is referenced.
```

Comment:
This seems to be copypasted from Norm 1.0. AFAIU `fk` value can only be a Model type, and the whole thing works based on the assumption that `fld` is integer and thus can be used as a foreign key to the referenced model `id` field, which is integer too.

Reply:
Fixed it.